<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouriesse Palace</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/iconfinder_Crown_7176689.png</normaloff>../../Downloads/iconfinder_Crown_7176689.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1001</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageLogin">
     <widget class="QLabel" name="labelBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>681</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/res/final_bg.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>470</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-bottom: 7px;
border: none;
background-color: rgba(255, 255, 255, 0);
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditID">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>350</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-bottom: 5px;
border: none;
background-color: rgba(255, 255, 255, 0);
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageHome">
     <widget class="QStackedWidget" name="stackedWidgetLeft">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>-1</y>
        <width>491</width>
        <height>681</height>
       </rect>
      </property>
      <widget class="QWidget" name="pageList">
       <widget class="QLabel" name="labelTitleList">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>261</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Gestion des Clients</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAddClient">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonDelClient">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelClientCount">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Clients total : 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidgetDrag">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>471</width>
          <height>581</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px;</string>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageView">
       <widget class="QLabel" name="labelTitleClientInfo">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>481</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Informations Client</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelBgClientInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>461</width>
          <height>581</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelLogoClientInfo">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>100</y>
          <width>131</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Desktop/res/user-max.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>270</y>
          <width>421</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Flavient DELACROIX</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoID">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>421</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>#5698247832</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="Line" name="lineSpaceInfo">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>360</y>
          <width>16</width>
          <height>171</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoPhone">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Téléphone: 06.45.85.63.21</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoMail">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>400</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mail: flavient.delacroix@gmail.com</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoAddress">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adresse: 8 rue Masseau</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoCity">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>450</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>75000 Paris</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoPay">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>500</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mode de règlement: CB</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfoAssoc">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>360</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Associées: 1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonInfoUpdate">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>570</y>
          <width>231</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Mettre à Jour</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonInfoBack">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>610</y>
          <width>231</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;- Retour</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageAdd">
       <widget class="QPushButton" name="pushButtonAddCancel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>610</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetRight">
      <property name="geometry">
       <rect>
        <x>509</x>
        <y>-1</y>
        <width>491</width>
        <height>681</height>
       </rect>
      </property>
      <widget class="QWidget" name="pageSelect">
       <widget class="QLabel" name="labelTitleReserv">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>311</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Gestion des Réservations</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidgetReserv">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>471</width>
          <height>581</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="gridSize">
         <size>
          <width>156</width>
          <height>158</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="itemAlignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageFocus">
       <widget class="ListWidget" name="listWidgetDrop">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>463</width>
          <height>182</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(&quot;C:/Users/Tanguy/Desktop/res/drop_zone.png&quot;);
background-position: center;
border: 0px;</string>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidgetReservation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>461</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonBackHome">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>640</y>
          <width>461</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;- Retour</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTitleFocus">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>20</y>
          <width>451</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Réservation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelCountReserv">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>60</y>
          <width>451</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Il reste 0 réservations de disponible</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>listWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
